snippets/swift-testing.code-snippets

```diff
@@ -0,0 +1,53 @@
+{
+  "@Test": {
+    "prefix": "@Test",
+    "body": [
+      "@Test func ${1:name}() async throws {",
+      "\t$2",
```

Comment:
To keep this consistent with standard snippets perhaps it should be ${2:statements}
Similarly for all the other test function snippets

Reply:
Done